<?xml version="1.0" encoding="UTF-8"?>
<!-- SPDX-FileCopyrightText: Copyright (c) Kitware Inc. -->
<!-- SPDX-License-Identifier: BSD-3-Clause -->
<ui version="4.0">
 <class>MultiBlockInspectorWidget</class>
 <widget class="QWidget" name="MultiBlockInspectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiBlock Inspector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,0,0,0,0,0,0,0,0,0">
   <property name="spacing">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="container" native="true">
     <layout class="QVBoxLayout" name="containerLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="extractBlocks">
     <property name="text">
      <string>Extract Blocks</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="showHints">
       <property name="toolTip">
        <string>Show/Hide legend</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../pqComponents.qrc">
         <normaloff>:/pqWidgets/Icons/pqHelp.svg</normaloff>:/pqWidgets/Icons/pqHelp.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelHeader">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>State Legend:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_stateDisabled">
     <item>
      <widget class="QLabel" name="iconStateDisabled">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="pixmap">
        <pixmap resource="../pqComponents.qrc">:/pqWidgets/Icons/pqStateDisabled.svg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelStateDisabled">
       <property name="text">
        <string>At least one property is disabled because no blocks are selected.</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_stateRepresentationInherited">
     <item>
      <widget class="QLabel" name="iconStateRepresentationInherited">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="pixmap">
        <pixmap resource="../pqComponents.qrc">:/pqWidgets/Icons/pqStateRepresentationInherited.svg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelStateRepresentationInherited">
       <property name="text">
        <string>At least one property is inherited from the representation.</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_stateBlockInherited">
     <item>
      <widget class="QLabel" name="iconStateBlockInherited">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="pixmap">
        <pixmap resource="../pqComponents.qrc">:/pqWidgets/Icons/pqStateBlockInherited.svg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelStateBlockInherited">
       <property name="text">
        <string>At least one property is inherited from block(s).</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_stateMixedInherited">
     <item>
      <widget class="QLabel" name="iconStateMixedInherited">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="pixmap">
        <pixmap resource="../pqComponents.qrc">:/pqWidgets/Icons/pqStateMixedInherited.svg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelStateMixedInherited">
       <property name="text">
        <string>At least one property is inherited from block(s) and the representation.</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_stateSet">
     <item>
      <widget class="QLabel" name="iconStateSet">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="pixmap">
        <pixmap resource="../pqComponents.qrc">:/pqWidgets/Icons/pqStateSet.svg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelStateSet">
       <property name="text">
        <string>At least one property is set in block(s).</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_stateSetAndRepresentationInherited">
     <item>
      <widget class="QLabel" name="iconStateSetAndRepresentationInherited">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="pixmap">
        <pixmap resource="../pqComponents.qrc">:/pqWidgets/Icons/pqStateSetAndRepresentationInherited.svg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelStateSetAndRepresentationInherited">
       <property name="text">
        <string>At least one property is set in block(s) and inherited from the representation.</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_stateSetAndBlockInherited">
     <item>
      <widget class="QLabel" name="iconStateSetAndBlockInherited">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="pixmap">
        <pixmap resource="../pqComponents.qrc">:/pqWidgets/Icons/pqStateSetAndBlockInherited.svg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelStateSetAndBlockInherited">
       <property name="text">
        <string>At least one property is set in block(s) and inherited from block(s).</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_stateSetAndMixedInherited">
     <item>
      <widget class="QLabel" name="iconStateSetAndMixedInherited">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="pixmap">
        <pixmap resource="../pqComponents.qrc">:/pqWidgets/Icons/pqStateSetAndMixedInherited.svg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelStateSetAndMixedInherited">
       <property name="text">
        <string>At least one property is set in block(s) and inherited from block(s) and the representation.</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pqComponents.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>showHints</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelHeader</receiver>
   <slot>setVisible(bool)</slot>
  </connection>
  <connection>
   <sender>showHints</sender>
   <signal>toggled(bool)</signal>
   <receiver>iconStateDisabled</receiver>
   <slot>setVisible(bool)</slot>
  </connection>
  <connection>
   <sender>showHints</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelStateDisabled</receiver>
   <slot>setVisible(bool)</slot>
  </connection>
  <connection>
   <sender>showHints</sender>
   <signal>toggled(bool)</signal>
   <receiver>iconStateRepresentationInherited</receiver>
   <slot>setVisible(bool)</slot>
  </connection>
  <connection>
   <sender>showHints</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelStateRepresentationInherited</receiver>
   <slot>setVisible(bool)</slot>
  </connection>
  <connection>
   <sender>showHints</sender>
   <signal>toggled(bool)</signal>
   <receiver>iconStateBlockInherited</receiver>
   <slot>setVisible(bool)</slot>
  </connection>
  <connection>
   <sender>showHints</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelStateBlockInherited</receiver>
   <slot>setVisible(bool)</slot>
  </connection>
  <connection>
   <sender>showHints</sender>
   <signal>toggled(bool)</signal>
   <receiver>iconStateMixedInherited</receiver>
   <slot>setVisible(bool)</slot>
  </connection>
  <connection>
   <sender>showHints</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelStateMixedInherited</receiver>
   <slot>setVisible(bool)</slot>
  </connection>
  <connection>
   <sender>showHints</sender>
   <signal>toggled(bool)</signal>
   <receiver>iconStateSet</receiver>
   <slot>setVisible(bool)</slot>
  </connection>
  <connection>
   <sender>showHints</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelStateSet</receiver>
   <slot>setVisible(bool)</slot>
  </connection>
  <connection>
   <sender>showHints</sender>
   <signal>toggled(bool)</signal>
   <receiver>iconStateSetAndRepresentationInherited</receiver>
   <slot>setVisible(bool)</slot>
  </connection>
  <connection>
   <sender>showHints</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelStateSetAndRepresentationInherited</receiver>
   <slot>setVisible(bool)</slot>
  </connection>
  <connection>
   <sender>showHints</sender>
   <signal>toggled(bool)</signal>
   <receiver>iconStateSetAndBlockInherited</receiver>
   <slot>setVisible(bool)</slot>
  </connection>
  <connection>
   <sender>showHints</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelStateSetAndBlockInherited</receiver>
   <slot>setVisible(bool)</slot>
  </connection>
  <connection>
   <sender>showHints</sender>
   <signal>toggled(bool)</signal>
   <receiver>iconStateSetAndMixedInherited</receiver>
   <slot>setVisible(bool)</slot>
  </connection>
  <connection>
   <sender>showHints</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelStateSetAndMixedInherited</receiver>
   <slot>setVisible(bool)</slot>
  </connection>
 </connections>
</ui>
